doc/bc/changes-6.11.md

```diff
@@ -0,0 +1,20 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+- EZP-26885: Field Type external storage
+  - Field Type external storage gets directly injected with a proper external storage gateway.
```

Comment:
yes I think so

we almost always first deprecate, and in a later release change or remove ;)